features/step_definitions/asset_host_steps.rb

```diff
@@ -38,7 +38,7 @@
   assert_not_nil Capybara.page.source.match js_regexp
 end
 
-Then /^provided assets shouldn't be loaded from the asset host$/ do
+Then /^cdn assets shouldn't be loaded from the asset host$/ do
   # When no CDN is set, we expect to find relative paths to reusable assets
   js_regexp =  %r{src="/dev-portal-assets/.*?\.js"}
```

Comment:
btw in all negative tests I would prefer also to have a positive part. For example make sure that assets are loaded just with a different path/url. Because if we look at the wrong page for some reason, the negative test may pass but because page was wrong, not because functionality was right.